src/transformStateWithClones.js

```diff
@@ -65,7 +65,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  const newObj = { ...state };
+
+  for (const obj in transforms) {
+    if (transforms[obj].operation === 'addProperties') {
+      for (const key in transforms[obj].properties) {
+        newObj[key] = transforms[obj].properties[key];
+      }
+    }
+
+    if (transforms[obj].operation === 'removeProperties') {
```

Comment:
For my opinion it would be better to use if else or switch case

Reply:
yes, use switch case